src/main/java/htsjdk/samtools/SAMSequenceRecord.java

```diff
@@ -43,9 +49,12 @@
 {
     public static final long serialVersionUID = 1L; // AbstractSAMHeaderRecord implements Serializable
     private String mSequenceName = null; // Value must be interned() if it's ever set/modified
+    @XmlAttribute(name = "alternative_sequece_names")
+    private Set<String> mAlternativeSequenceName = new LinkedHashSet<>();
```

Comment:
This should be handled the same as the other attributes, stored in the attributes map.  It means doing annoying string processing when you add a single alias to an existing set of aliases, but I think making it consistent is worthwhile